src/services/Filter.php

```diff
@@ -164,6 +164,10 @@ public static function body(?string $input = null): string
         // allow any image size, see #3800
         $config->set('CSS.MaxImgLength', null);
         $config->set('HTML.MaxImgLength', null);
+        // allow 'data-table-sort' attribute to indicate that a table shall be sortable by js
+        if ($def = $config->maybeGetRawHTMLDefinition()) {
```

Comment:
why is this needed if it is in the config above?

Reply:
This is the way htmlpurifier needs to be configured. Compare to http://htmlpurifier.org/docs/enduser-customize.html.
I tried without the additional lines but it does not work.
Basically HTMLPurifier only supports the 'default' attributes but any custom attribute needs more information. E.g.: what are the allowed values of that attribute.